cmd.js

```diff
@@ -36,6 +36,10 @@ files.forEach(f => {
     })
   })
 })
-process.on('SIGTERM', killCommand) 
-process.on('SIGINT', killCommand) 
-process.on('uncaughtException', killCommand) 
+function die() {
+  killCommand()
```

Comment:
The `killCommand` call might not be necessary.

From [the docs](https://nodejs.org/api/child_process.html#child_process_options_detached):
> If the parent's `stdio` is inherited, the child will remain attached to the controlling terminal.